benchmarks/common.py

```diff
@@ -104,7 +105,8 @@ def print_summary(filename):
                     f"gmean={gmean(cdata):.2f}x mean={cdata.mean():.2f}x",
                 )
         except Exception:
-            pass
+            traceback.print_exc(file=sys.stderr)
+            logging.warning("ignoring failed summary and continuing...")
```

Comment:
```suggestion
            logging.warning("ignoring failed summary and continuing...", exc_info=True)
```
This will make logging print the exception so you don't need `traceback.print_exc(file=sys.stderr)`